pkg/execution/runner/runner.go

```diff
@@ -310,13 +311,67 @@ func (s *svc) handleMessage(ctx context.Context, m pubsub.Message) error {
 	return errs
 }
 
+// InvokeFunction is a helper method which loads all available functions, checks
+// the incoming event and returns the function to be invoked via the RPC invoke event,
+// or nil if a function is not being invoked.
+func InvokeFunction(ctx context.Context, evt event.Event, fl cqrs.ExecutionLoader) (*inngest.Function, error) {
```

Comment:
nit:
this sounds more like a `FindFunction` then `Invoke`.
it's not actually invoking anything per se.

Reply:
Let's call this `FindInvokedFunction` - it's really only finding functions if the event is an invoke event.  Thanks for the feedback!